### DISCUSSION TITLE
Is is actually the fastest web server?

### DISCUSSION THREAD

**Discussion by gund** [honojs/hono#3237] [Q&A]
Are you claiming that this web server written in TS is actually faster then the [one written in C++](https://github.com/uNetworking/uWebSockets.js)?
Even according to [this list](https://github.com/SaltyAom/bun-http-framework-benchmark#results) hono is not on the top 1 place...
What am I missing?

**Comment by NicoPlyley** [comment]
I'm not sure what you're seeing but I don't believe Hono makes the claim that they are the fastest, only that it is fast.

**Comment by gund** [comment]
Hmm you might be right, but by reading the readme "Hono - means flame🔥 in Japanese - is a small, simple, and ultrafast web framework built on Web Standards" - "ultrafast web framework" gave me the impression that it's the fastest one =)

**Reply by adworacz** [reply]
Just because something claims it is fast, doesn't make it the fastest. The same concept applies to sports cars - they're all fast, but they're not all the fastest.

If you are assuming that Hono is the "fastest" simply because it claims to be fast, that's your mistake and not Hono's.

**Reply by gund** [reply]
I'm just pointing out that "ultrafast web framework" makes the impression that it's the fastest - fast is fast but ultrafast is faster then fast isn't it?
Simply suggesting to dial down a bit the tone so that it's not confusing people, unless it's the intent of course 😉
